refine/plugins/custom-layout/extend.js

```diff
@@ -0,0 +1,21 @@
+const base = {
+    _app: {
+        refineProps: [
+            "Title={Title}",
+            "Header={Header}",
```

Comment:
Burası i18n seçili değilse gelmesi lazım, yoksa i18n'den de Header gelicek iki tane refineProps={Header} olmuş olur

Reply:
Bunu superplate bu objeleri [şurda](https://github.com/pankod/superplate/blob/master/src/Helper/plugin/index.ts#L84) merge'lerken yapsa daha iyi olmaz mı? 
`merge.all` için `arrayMerge` option'ı var, burda array'i uniq yapan bir method olabilir.
Mesela:
- ```arrayMerge: (destination, source) => [...new Set([...destination, ...source])]```
yalnız bunun için tsconfig'e `"downlevelIteration": true,` eklemek gerekiyor.
- lodash'dan `uniq` kullanılabilir.